### PR TITLE
feat: show plugin permissions on install

### PR DESCRIPTION
**Pull request by luciorubeens**
## Summary

https://app.clickup.com/t/ck1v0h

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Changes requested as discussed on Slack.

The permission modal only shows when installing via 3 dot menu.

**Review by samharperpittam — APPROVED** [review]

File: src/domains/plugin/components/InstallPlugin/InstallPlugin.tsx [hunk 1]
```diff
@@ -1,81 +1,53 @@
...
-
-	const handleNext = () => {
-		setActiveStep(activeStep + 1);
-	};
+	const { installPlugin } = usePluginManagerContext();
+
+	const handleDownload = useCallback(async () => {
+		try {
+			await installPlugin(plugin.id, repositoryURL);
+			toasts.success(`The plugin "${plugin.title}" was successfully installed`);
```
Comment (ItsANameToo) [inline src/domains/plugin/components/InstallPlugin/InstallPlugin.tsx:27]: we should move these toasts to language files too

File: src/domains/plugin/pages/PluginsCategory/PluginsCategory.tsx [hunk 2]
```diff
@@ -153,6 +142,7 @@ export const PluginsCategory = ({ title, description, initialViewType }: Plugins
 			</Page>
 
 			<InstallPlugin
+				plugin={{}}
```
Comment (ItsANameToo) [inline src/domains/plugin/pages/PluginsCategory/PluginsCategory.tsx:145]: what's this empty object for?

Comment (luciorubeens) [reply src/domains/plugin/pages/PluginsCategory/PluginsCategory.tsx:145]: This component is not used anywhere yet.

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**